extensions/amp-story/1.0/test/test-amp-story-cta-layer.js

```diff
@@ -63,4 +74,43 @@ describes.realWin('amp-story-cta-layer', {
     });
   });
 
+  it('should not allow a cta layer on the first page', () => {
+    // Setup: create story with two pages.
+    const ampStoryEl = win.document.createElement('amp-story');
+    win.document.body.appendChild(ampStoryEl);
+    createPages(ampStoryEl, 2, ['cover', 'next-page']);
```

Comment:
I'm not a big fan of having to replicate this logic here.
Would that work with a more minimal structure? I believe we don't even need the ``amp-story`` tag

What do you think if something like

````
<amp-story-page>
  <cta-layer></cta-layer>
<amp-story-page>
````

and then

````
<amp-story-page><amp-story-page>
<amp-story-page>
  <cta-layer></cta-layer>
<amp-story-page>
````

Without even having to instantiate the pages?